src/components/ProfileEditPOWnamePage.js

```diff
@@ -0,0 +1,153 @@
+import React from "react"
+import { useState } from "react"
+import { useAuthActions, useAuthState } from "../auth"
+import { navigate } from "gatsby"
+import {
+  AppBar,
+  Button,
+  Toolbar,
+  IconButton,
+  TextField,
+  Paper,
+  Typography,
+  makeStyles,
+} from "@material-ui/core"
+import ArrowBackIcon from "@material-ui/icons/ArrowBack"
+
+// Is this CSS?
+// Is this using classes in the DOM?
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: "50rem",
+    margin: "0 auto",
+  },
+  form: {
+    margin: theme.spacing(3),
+    padding: theme.spacing(3),
+    maxWidth: "35em",
+  },
+  helperText: {
+    padding: `0 ${theme.spacing(1)}`,
+  },
+  appBar: {
+    borderTop: `4px solid ${theme.palette.primary.main}`,
+    borderBottom: `1px solid ${theme.palette.grey[200]}`,
+  },
+  toolbar: {
+    width: "100%",
+    maxWidth: "55rem",
+    margin: "0 auto",
+  },
+  title: {
+    flexGrow: 1,
+  },
+}))
+
+//async somewhere here look at
+// Line 83 in UserForm.js
+
+const ProfileEditnamePage = () => {
+  const classes = useStyles()
+  const { updateUser } = useAuthActions()
+  const { user } = useAuthState()
+  const currentUsername = user.username
+  const [ussername, setUssername] = React.useState("")
+  const isLowerCase = ussername === ussername.toLowerCase()
+  const errorDodd = isLowerCase ? null : "Your username must be lowecase" // from userbase
+  const [error, setError] = useState(false)
+
+  const createNewPOWname = async (event) => {
+    // Go GET that event and stop the page from fully and naughtily refreshing
+    event.preventDefault()
+
+    // LISTEN for CustomerName
+    const newUsername = event.target.elements.POWnameInput.value
+
+    // ESCAPE that ({username: myNewPOWname}) to DanielV's Userbase.com
+
+    const result = await updateUser({ username: newUsername })
+
+    if (result.error) {
+      setError(result.error)
+    } else {
+      setError(false)
+    }
+
+    // EVADE back to (`/profile`) by calling the navigate from Gatsby
+    navigate(`/profile`)
+  }
+
+  const createReset = (event) => {
+    event.preventDefault()
+    navigate(`/profile`)
+  }
+  function handleChange(event) {
+    setUssername(event.target.value)
+  }
+  return (
+    <div className={classes.root}>
+      <Toolbar />
+      <Paper
+        component="form"
+        onSubmit={createNewPOWname}
+        onReset={createReset}
+        className={classes.form}
+      >
+        <label htmlFor="POWnameInput">
```

Comment:
I would rather is be named "UsernameInput".

Reply:
UsernameInput